images.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"github.com/k0kubun/go-ansi"
+	"github.com/schollz/progressbar/v3"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"slices"
+)
+
+type ImageInfo struct {
+	Path string
+	Info fs.FileInfo
+}
+
+/**
+ * Determine if the given file looks like an image
+ */
+func isImage(fileInfo fs.FileInfo) bool {
+	extensions := []string{".jpg", ".jpeg", ".png", ".gif", ".JPG", ".JPEG", ".PNG", ".GIF"}
```

Comment:
I think having a single set of lowercase extensions and then calling `strings.ToLower(extension)` would be better than trying to predict all possible cases.